server/app/driver/driver.py

```diff
@@ -19,17 +19,15 @@ def genes(self):
         pass
 
     @abstractmethod
-    def filter_cells(self, filter):
+    def filter_dataframe(self, filter):
         """
-        Filter cells from data and return a subset of the data
-        A filter is a dictionary where the key is a metadatata category
-        Value is dictionary
-            value_type: int, float, string
-            variable_type: continuous, categorical
-            query: filter value, for categorical [val1, val2], for continuous {min: x, max:y}
-        Filters are combined with the and operator
-        :param filter:
-        :return: filtered dataframe
+         Filter cells from data and return a subset of the data. They can operate on both obs and var dimension with
+         indexing and filtering by annotation value. Filters are combined with the and operator.
+         See REST specs for info on filter format:
+         https://docs.google.com/document/d/1Fxjp1SKtCk7l8QP9-7KAjGXL0eldi_qEnNT0NmlGzXI/edit#heading=h.8qc9q57amldx
```

Comment:
I'm curious why this isn't just a pointer to the swagger spec you have in `server/app/rest_api/rest.py`

Reply:
This is a new spec. rest.py is obsolete at this point in the v2.0 refactor. I'll make a note to update that when it is up to date